interfaces/system_key_test.go

```diff
@@ -105,6 +104,11 @@ func (s *systemKeySuite) TestInterfaceWriteSystemKey(c *C) {
 	c.Check(string(systemKey), Equals, fmt.Sprintf(`{"version":1,"build-id":"%s","apparmor-features":%s,"apparmor-parser-mtime":%s,"apparmor-parser-features":%s,"nfs-home":%v,"overlay-root":%q,"seccomp-features":%s}`, buildID, apparmorFeaturesStr, apparmorParserMtime, apparmorParserFeaturesStr, nfsHome, overlayRoot, seccompActionsStr))
 }
 
+func (s *systemKeySuite) TestInterfaceWriteSystemKey(c *C) {
+	s.testInterfaceWriteSystemKey(c, false)
+	s.testInterfaceWriteSystemKey(c, true)
```

Comment:
can't we have two tests instead of two tests in one basically?

Reply:
Right, makes sense, will do.